ampd/src/broadcaster/mod.rs

```diff
@@ -349,7 +347,8 @@ where
 }
 
 fn decode_base_account(account: Any) -> Result<BaseAccount, Error> {
-    BaseAccount::decode(&account.value[..])
+    account
+        .to_msg()
```

Comment:
to_msg gives you a `BaseAccount`?

Reply:
Yes. They made "Any::from_msg` to convert to Any, and `.to_msg` to convert from Any to a concrete type.